src/site/tools/pub/publishing.markdown

```diff
@@ -7,29 +7,81 @@ title: "Publishing a Package"
 
 # {{ page.title }}
 
-[Pub](/tools/pub) isn't just for using other people's packages. It also allows you to share
-your packages with the world. Once you've written some useful code and you want
-everyone else to be able to use it, just run:
+[Pub](/tools/pub) isn't just for using other people's packages.
+It also allows you to share your packages with the world. If you have a useful
+project and you want others to be able to use it, use the `pub publish`
+command. For your first time around, you can perform a dry run like so:
+
+{% prettify sh %}
```

Comment:
Change this to none? (It's indented differently from the 'none' code, which looks weird.)